tests/serialize/t10-ref.ttl

```diff
@@ -1,47 +1,52 @@
 @prefix dc: <http://purl.org/dc/elements/1.1/>.
+@prefix sch: <http://www.w3.org/ns/pim/schedule#>.
 @prefix c: <https://www.w3.org/People/Berners-Lee/card#>.
 @prefix XML: <http://www.w3.org/2001/XMLSchema#>.
-@prefix sch: <http://www.w3.org/ns/pim/schedule#>.
 @prefix sp: <http://www.w3.org/ns/pim/space#>.
 @prefix det: <http://linkeddata.github.io/app-schedule/details.ttl#>.
 @prefix rd: <http://www.w3.org/2000/01/rdf-schema#>.
 @prefix ic: <http://www.w3.org/2002/12/cal/ical#>.
 @prefix n0: <http://xmlns.com/foaf/0.1/>.
 
-<#event>
-    dc:author    c:i;
-    dc:created    "2015-09-15T11:20:05Z"^^XML:dateTime;
-    a    sch:SchedulableEvent;
-    sp:inspiration    det:event1.
-<#event1>
-    dc:author    <#id1442316021316>;
-    dc:title    "TAG 2016 Q1 Meeting ";
-    rd:comment    "Three day meeting, starting date:";
-    ic:location    "Melbourne Australia";
-    sch:availabilityOptions
-       sch:YesNoMaybe;
+:event
```

Comment:
In the test suite, yes, as otherwise we build in knowledge of what directory the test is run in, an that breaks when someone else runs it, like Travis.
In the systems we build on top of the library, no we should use Turtle not RDF/XML.
In the systems random developers, the subset who decide for some reason to use RDF/XML, then it is hard to say which will be more likely to cause unexpected problems.  After all by default we do in XML attributes use relative URIs (I assume) so they will have to pass a base URI to their parser.  It is just if their parser does not use it on the namespace URIs.   I guess the Jena folks could have systems like that, as IIRC they didn't use the rule of always serializing relative to a base (so they would get massive issues when they moved systems).  Yes, I suppose to be safe we should flip the flag back and pass it in the test suite program data.js (which is an early version of rabel the command line tool)